ee/clickhouse/queries/funnels/test/test_funnel_persons.py

```diff
@@ -366,13 +352,13 @@ def test_first_step_breakdowns_with_multi_property_breakdown(self):
         self.assertCountEqual([val["id"] for val in results], [person1.uuid, person2.uuid])
 
         _, results = ClickhouseFunnelActors(
-            filter.with_data({"funnel_step_breakdown": "Chrome"}), self.team
+            filter.with_data({"funnel_step_breakdown": ["Chrome", "95"]}), self.team
```

Comment:
@pauldambra was the intent here to allow filtering by a "part" of the entire breakdown value?

Not quite sure that works well, since the frontend would only have counts for the entire array that you can then query. Plus, leads to issues like

```
hasAll(['Mac', ''], ['', '']) = True
```
where what you're searching for and what you get are completely different.

(related change: https://github.com/PostHog/posthog/pull/10062/files#diff-c557ef3a0404400e4393bf21798be603d8e1f578db4d8532ca4662364773b710R558 )